File: StaticSharp/Modifiers/BackdropGrayscale/BackdropGrayscale.js
StaticSharpClass("StaticSharp.BackdropGrayscale", (modifier, element) => {
    StaticSharp.AbstractBackdropFilter(modifier, element)

    modifier.Reactive = {
        Amount: 1
    }

    modifier.getFilter = function () {
        return `grayscale(${modifier.Amount})`
    }
})

File: StaticSharp/Modifiers/BackdropGrayscale/BackdropGrayscale.cs
using StaticSharp.Gears;
using System.Runtime.CompilerServices;

namespace StaticSharp {

    public interface JBackdropGrayscale : JAbstractBackdropFilter {
        double Amount { get; set; }
    }

    [ConstructorJs]
    public partial class BackdropGrayscale : AbstractBackdropFilter {
        
    }


}


